### PR TITLE
Fix(CX-3320): SWA Confirmation Page fixes

### PR DESCRIPTION
**Pull request by lordkiz**
The type of this PR is: **Fix**

This PR solves [CX-3320]

### Description
- This PR addresses the issues raised following QA. QA Notes in the comment of the linked Jira ticket.

Screenshot 2023-01-16 at 20 20 21
Screenshot 2023-01-16 at 20 19 56

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CX-3320]: https://artsyproduct.atlassian.net/browse/CX-3320?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Apps/Consign/Routes/SubmissionFlow/ThankYou/ThankYou.tsx [hunk 1]
```diff
@@ -76,25 +80,22 @@ export const ThankYou: React.FC = () => {
...
+          size="large"
+          variant="primaryBlack"
+          onClick={submitAnotherWork}
+        >
+          Submit Another Work
+        </Button>
         <Spacer x={[0, 2]} y={[2, 0]} />
         <Button
-          width="100%"
+          width={["100%", "50%"]}
```
Comment (dariakoko) [inline src/Apps/Consign/Routes/SubmissionFlow/ThankYou/ThankYou.tsx:98]: The buttons' size is different. Is it intentional?
`width={["100%", "80%", "50%"]} ` / `width={["100%", "50%"]}`

File: src/Apps/Consign/Routes/SubmissionFlow/SubmissionLayout.tsx [hunk 2]
```diff
@@ -19,13 +18,11 @@ export const SubmissionLayout: React.FC = ({ children }) => {
       <Spacer y={4} />
 
       <AppContainer>
-        <HorizontalPadding>
-          <RouterLink to="/sell" display="block">
-            <ArtsyLogoBlackIcon display="block" />
-          </RouterLink>
+        <RouterLink to="/sell" display="block">
+          <ArtsyLogoBlackIcon display="block" />
+        </RouterLink>
```
Comment (damassi) [inline src/Apps/Consign/Routes/SubmissionFlow/SubmissionLayout.tsx:23]: @dzucconi - should this file be our new layouts pattern?

File: src/Apps/Consign/Routes/SubmissionFlow/ThankYou/ThankYou.tsx (hunk 1 above)
Comment (lordkiz) [reply src/Apps/Consign/Routes/SubmissionFlow/ThankYou/ThankYou.tsx:98]: @dariakoko . Yes it is. 100 for mobile, 80 for pads, 50 for desktop.

Comment (dariakoko) [reply src/Apps/Consign/Routes/SubmissionFlow/ThankYou/ThankYou.tsx:98]: I mean "Submit Another Work" button has three size options and "View Artwork in My Collection" has just two

File: src/Apps/Consign/Routes/SubmissionFlow/SubmissionLayout.tsx (hunk 2 above)
Comment (dzucconi) [reply src/Apps/Consign/Routes/SubmissionFlow/SubmissionLayout.tsx:23]: Really, these routes should be using the `LogoOnly` layout. The logo shouldn't be linked to `/sell` it should always go to the homepage. Immediate context should be represented with the `TopContextBar`. 🤷 

Which is to say: I don't think this layout warrants an actual layout; but I do think it should be reconsidered.

File: src/Apps/Consign/Routes/SubmissionFlow/ThankYou/ThankYou.tsx (hunk 1 above)
Comment (dzucconi) [reply src/Apps/Consign/Routes/SubmissionFlow/ThankYou/ThankYou.tsx:98]: Buttons are never really percentage widths other than `100%`, they typically should be sized to occupy column spans in our grid system.

Comment (lordkiz) [reply src/Apps/Consign/Routes/SubmissionFlow/ThankYou/ThankYou.tsx]: @dariakoko That is actually the container. The buttons are the children here. But I have refactored to use Columns.

Comment (lordkiz) [reply src/Apps/Consign/Routes/SubmissionFlow/ThankYou/ThankYou.tsx]: Thanks @dzucconi

File: src/Apps/Consign/Routes/SubmissionFlow/SubmissionLayout.tsx (hunk 2 above)
Comment (lordkiz) [reply src/Apps/Consign/Routes/SubmissionFlow/SubmissionLayout.tsx:23]: I agree, it seems weird that a logo would redirect to a page other than the homepage, but Design was specific about this. I only had to remove the horizontal padding.

**Review by dariakoko — APPROVED** [review]
Thank you for the changes!

**Pull request merged**